cmd/e2e_test.go

```diff
@@ -10,11 +10,12 @@ import (
 	"github.com/onsi/ginkgo/v2"
 	"github.com/onsi/ginkgo/v2/types"
 	"github.com/onsi/gomega"
-	_ "github.com/redhat-appstudio/e2e-tests/tests/build"
```

Comment:
I can't imagine the changes to this file actually want to be merged, right?

Reply:
my bad , 
removed the comment from lines